app/javascript/gobierto_data/webapp/components/commons/InfoList.vue

```diff
@@ -1,24 +1,140 @@
 <template>
-  <div>
-    <InfoListElement
-      v-for="(item, index) in items"
+  <div class="pure-u-1 pure-u-lg-3-4 gobierto-data-layout-column">
+    <div class="gobierto-data-infolist">
+    </div>
+    <div
+      v-for="(item, index) in allDatasets"
       :key="index"
-      :item="item"
       class="gobierto-data-info-list-element"
-    />
+    >
+      <a
+        :href="'/datos/' + item.attributes.slug"
+        class="gobierto-data-title-dataset gobierto-data-title-dataset-big"
+        @click.prevent="nav(item.attributes.slug)"
+      >
+        {{ item.attributes.name }}
+      </a>
+      <div class="pure-g">
+        <div class="pure-u-1-2 gobierto-data-summary-header">
+          <div class="gobierto-data-summary-header-container">
+            <i
+              class="fas fa-clock"
+              style="color: var(--color-base); opacity: .25"
+            />
+            <span class="gobierto-data-summary-header-container-label">
+              {{ labelUpdated }}
+            </span>
+            <span class="gobierto-data-summary-header-container-text">
+              {{ item.attributes.data_updated_at | convertDate }}
+            </span>
+          </div>
+          <div
+            v-if="item.attributes.frequency[0].name_translations"
+            class="gobierto-data-summary-header-container"
+          >
+            <i
+              class="fas fa-calendar"
+              style="color: var(--color-base); opacity: .25"
+            />
+            <span class="gobierto-data-summary-header-container-label">
+              {{ labelFrequency }}
+            </span>
+            <template v-if="selectedLanguage === 'es' && item.attributes.frequency[0].name_translations !== undefined">
+              <span class="gobierto-data-summary-header-container-text">
+                {{ item.attributes.frequency[0].name_translations.es }}
+              </span>
+            </template>
+            <template v-else-if="item.attributes.frequency[0].name_translations !== undefined">
+              <span class="gobierto-data-summary-header-container-text">
+                {{ item.attributes.frequency[0].name_translations.ca }}
+              </span>
+            </template>
```

Comment:
I believe this could be solved in one line:

`{{ item.attributes.frequency[0].name_translations[I18n.locale] }}`